liboptv/include/sortgrid.h

```diff
@@ -0,0 +1,39 @@
+/* Utilities for handling sortgrid */
+
+#ifndef SORTGRID_H
+#define SORTGRID_H
+
+#include "calibration.h"
+#include "parameters.h"
+#include "tracking_frame_buf.h"
+#include "imgcoord.h"
+#include "multimed.h"   
+#include "trafo.h"
+#include "vec_utils.h"
+#include <stdio.h>
+
+typedef struct
+{
+  int pnr;
+  vec3d pos;
+} coord_3d;
```

Comment:
Sortgrid knows nothing about the files. In a multiplane calibration you read all the known points (fix) files and concatenate them to one list, then you do the same with the detected points (crd) files. Then sortgrid matches one to the other. You are free to number the known points any way you like. This is what I did in my multiplane code and it works great.